### PR TITLE
Split Stanford's Django Settings

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

**Comment by stvstnfrd** [comment]
Please review: at least one of @caesar2164 @kluo @caseylitton @dcadams 
CC/FYI: everyone

**Comment by stvstnfrd** [comment]
retest this please

File: lms/envs/common.py [hunk 1]
```diff
@@ -259,14 +246,14 @@
...
-    # Grade calculation and response report requests started from the new instructor dashboard will write
-    # CSV files to S3 and give links for downloads.
+    # Grade calculation started from the new instructor dashboard will write
+    # grades CSV files to S3 and give links for downloads.
     'ENABLE_S3_GRADE_DOWNLOADS': False,
 
     # whether to use password policy enforcement or not
     'ENFORCE_PASSWORD_POLICY': True,
 
-    # Give course staff unrestricted access to grade and student responses downloads (if set to False,
```
Comment (caesar2164) [inline lms/envs/common.py]: why did we fork comments???

**Comment by caesar2164** [comment]
NICE!

I'm 👍

File: lms/envs/common.py (hunk 1 above)
Comment (stvstnfrd) [inline lms/envs/common.py]: Who knows...
It had been there for a whiiile...

**Pull request merged**